Tracking/src/Tracking/Reco/TruthSeedProcessor.cxx

```diff
@@ -57,9 +57,13 @@ void TruthSeedProcessor::configure(framework::config::Parameters& parameters) {
   inflate_factors_ = parameters.getParameter<std::vector<double>>(
       "inflate_factors", {10., 10., 10., 10., 10., 10.});
 
-  // In tracking frame
+  // In tracking frame:  MG where do these numbers come from?
```

Comment:
These numbers come from an approximation of where the beam generator places the beam (without smearing) just upstream of all detector components. `-880.1` is just upstream of TriggerPad1 in `z` and `-44` is approximately the bend in `x` imposed on the beam by the magnet (I think 4GeV but could be 8GeV). Then they are re-ordered to be in tracking coordinates.